BDArmory/Targeting/TargetSignatureData.cs

```diff
@@ -177,10 +177,10 @@ public Vector3 predictedPositionWithChaffFactor(float chaffEffectivity = 1f)
                     float distortionFactor = decoyFactor * UnityEngine.Random.Range(16f, 256f);
 
                     // Convert Float jammingFactor position bias and signatureFactor scaling to Vector3 position
-                    Vector3 signatureDistortion = distortionFactor * (vessel.GetSrfVelocity().normalized * -1f * jammingFactor + UnityEngine.Random.insideUnitSphere);
+                    Vector3 signatureDistortion = distortionFactor * (vessel.Velocity().normalized * -1f * jammingFactor + UnityEngine.Random.insideUnitSphere);
```

Comment:
This ought to be written as
```csharp
Vector3 signatureDistortion = distortionFactor * (UnityEngine.Random.insideUnitSphere - jammingFactor * vessel.Velocity().normalized);
```

I benchmarked `float` * `Vector3` operations a little while ago and it turns out that Unity doesn't optimise the order of operations (when it's valid to do so), so `a*b*V` (where `a` and `b` are `float` and `V` is a `Vector3`) is significantly faster than `a*V*b` or `V*a*b` since the former is doing a `float*float` and a `float*Vector3` operation, while the latter two are doing two `float*Vector3` operations.